minigalaxy/installer.py

```diff
@@ -334,3 +335,28 @@ def _mv(source_dir, target_dir):
             if os.path.exists(dst_file):
                 os.remove(dst_file)
             shutil.move(file_to_copy, destination_dir)
+
+
+# Some installers allow to choose game's language before installation (Divinity Original Sin or XCom EE / XCom 2)
+# "--list-languages" option returns "en-US", "fr-FR" etc... for these games.
+# Others installers return "French : Français" but disallow to choose game's language before installation
+def lang_install(installer):
+    languages = []
+    arg = ""
+    process = subprocess.Popen(["innoextract", installer, "--list-languages"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    output = stdout.decode("utf-8")
+
+    for line in output.split('\n'):
+        if not line.startswith(' -'):
+            continue
+        languages.append(line[3:])
+    for lang in languages:
+        if "-" in lang:  # lang must be like "en-US" only.
+            if Config.get("lang") == lang[0:2]:
+                arg = "--language={}".format(lang)
+                break
+            else:
+                arg = "--language=en-US"
```

Comment:
I would leave arg empty if the language isn't found. Then the default should be used, right?

Reply:
No. As i explained in #427, in arg is empty a random language is set.
For example with XCom Enemy Unknow, without "arg", the game has Asiatic language...
That's why i force to use en_US by default.